### ISSUE TITLE
3D Handles Problems

### ISSUE THREAD

**Issue by McCoyCOREUSA** [smeighan/xLights#3583]
PLEASE add option to make the 3D model handles smaller. The "normal" option is way too large for small models. In this screenshot with three nodes, it took my nearly an hour of zooming, trying to grab, etc., just to align the three nodes and move into position. I can't click on the center handle because it's mixed with the end handles. These need to be 1/4th of the size they are to be able to click them properly.

Also, can you add the option to make the outside handle tools (move arrows, rotate, etc.) to have extended arms? In this same screenshot, the arrows to move the model I would like to extend, because the green arrow is hidden within my house and I can't access it. If I can extend longer, then it would be extended above the roofline and useable again.
![2022-11-21_11-42-32](https://user-images.githubusercontent.com/104024793/203124986-55f677a8-7e00-4296-b8ae-d086efec0be2.jpg)

**Comment by AzGilrock** [comment]
I agree on the handle sizes I was dealing with that yesterday.  On the Mac the Normal and Large options were drawing the same size.  I'd work on it but I go out of town soon until Dec 2nd so it would be after I get back.

By the way to access inside your house click the checkbox in the parameters for the house model that will remove all the textures and only show the green mesh lines.

**Comment by AzGilrock** [comment]
Another issue if anyone looks at this before me is I was drawing a really big PolyLine model like triple the size of my house and I noticed the model handles are not centered properly.  You can see it when you are placing points the cursor is not aligned with the blue cross handle.  And if you draw a PolyLine and then edit the XML to change the "Poly Line" DisplayAs to "MultiPoint" it will draw a pixel inside every handle and you can see they are not centered correctly.

**Comment by keithsw1111** [comment]
I would need to see your layout. It is possible you are already really zoomed in (we have seen it before).

**Comment by AzGilrock** [comment]
You know you can turn off the Active flag for the house and hide it

**Comment by McCoyCOREUSA** [comment]
Gil, yes. In this case it's not accessing the extension handles, but accessing the three large square block handles that are all overlapping each other.

**Comment by McCoyCOREUSA** [comment]
Gil, ultimately that's what I ended up doing.

**Comment by keithsw1111** [comment]
A video is not what i meant ... can you use tools package logs and attach your layout.

**Comment by AzGilrock** [comment]
Here's an example of how the model handle is not centered on the point.  This is actually a screenshot of the new Multi-Point model I'm working on which has lines between the points which helps show where the intersection lands.
<img width="427" alt="Screenshot 2022-12-08 at 9 39 18 PM" src="https://user-images.githubusercontent.com/2024669/206625399-eca8b8f1-1cd5-4ef3-93b8-3db1502325ec.png">

**Comment by derwin12** [comment]
In next update.
